Feature/Building Filter (#87)
Esri/arcgis-maps-sdk-unreal-engine-samples: sample_project/Content/SampleViewer/Samples/BuildingFilter/README.md

# Building Filter

> Allows users to toggle the visibility of different levels, construction phases, disciplines, and categories within a building scene layer. This sample demonstrates how to explore building scene layers and filter based on various criteria.

Reviewer: ```suggestion
This sample demonstrates how to explore building scene layers. User can toggle the visibility of different levels, construction phases, disciplines, and categories within the building scene layer.
```

Reviewer: Not really important but I feel the main thing is exploring bsls

![image](https://github.com/user-attachments/assets/f35d80c0-4d71-4af6-b462-44ef3a5d2d66)

## How to use the sample (SampleViewer)
1. The SampleViewer scene should open by default. If it is not open, click the **SampleViewer** scene to open it.
2. Click play.
3. Using the UI, enter an APIKey in the input field to the top left.
4. Open the **Samples** drop-down and click **Building Filter** to open the level.
5. Use the UI to toggle visibility of different levels, construction phases, disciplines, and categories.

## How to use the sample (BuildingFilter Scene)
1. **Open the BuildingFilter Level:**
   - Navigate to the **BuildingFilter** level in your project and open it.
2. **Set the API Key:**
   - In the outliner, locate and select the **ArcGISMapActor**.
   - In the **Details** panel, find the field for the API key and enter your API key.
3. **Start the Simulation:**
   - Click the play button to start the simulation.
4. **Interact with the UI:**
   - **Service URL:**

>      - There is a space in the UI where you can add a service URL for a building scene layer. Enter the URL to load a specific building scene layer. You can also use a local filepath for the URL.

Reviewer: ```suggestion
     - There is a space in the UI where you can add a service URL for a building scene layer. Enter the URL to load a specific building scene layer. You can also use a local file path for the URL.
```
